app/src/main/java/fr/free/nrw/commons/actions/PageEditClient.java

```diff
@@ -5,6 +5,9 @@
 
 import io.reactivex.Observable;
 
+/**
+ * Uses the PageEditInterface to make edit calls to MW API
+ */
 public class PageEditClient {
```

Comment:
Most of the java docs for methods in this class are not descriptive. Either make it descriptive or remove it if you are not sure what it does. Please check the same for other classes as well. :)

Reply:
Sure I'll make them more descriptive

Reply:
Great. Ping here once you are done.

Reply:
@maskaravivek Sir done :)